__tests__/app.test.js

```diff
@@ -38,4 +38,37 @@ describe("GET /api/topics", () => {
       });
     });
   });
+});
+
+describe("GET /api/articles/:article_id", () => {
+  test("return 200 status code", () => {
+    return request(app).get("/api/articles/1").expect(200);
+  });
+  test("return an article object", () => {
+    return request(app).get("/api/articles/2").then(({body}) => {
+      expect(body.article).toContainAllKeys(["article_id", "title", "topic", "author", "body", "created_at", "votes", "article_img_url"]);
+    });
+  });
+  test("return 404 if article not found", () => {
+    return request(app).get("/api/articles/100000").expect(404);
+  });
+  test("return 500 if article_id is not an integer", () => {
```

Comment:
This should be a different status code, as we can't really test for a 500 internal server error. Think about what status code this should be when the the request is 'bad'.